cgmanifest.json

```diff
@@ -61,7 +61,7 @@
                 "Maven": {
                     "GroupId": "org.gradle",
                     "ArtifactId": "gradle-tooling-api",
-                    "Version": "6.4"
```

Comment:
please make sure whether there's breaking changes in gradle tooling api

Reply:
Yeah, actually we're using tooling-api 7.x, and I just forgot to change this file.